src/classes/com/sun/tdk/jcov/Merger.java

```diff
@@ -1011,12 +1013,14 @@ private int handleEnv_(EnvHandler opts) throws EnvHandlingException {
         }
 
         template = opts.getValue(DSC_TEMPLATE);
-        Utils.checkFileCanBeNull(template, "template filename", Utils.CheckOptions.FILE_EXISTS, Utils.CheckOptions.FILE_ISFILE, Utils.CheckOptions.FILE_CANREAD);
+        Utils.checkFileCanBeNull(template, "template filename",
+                Utils.CheckOptions.FILE_EXISTS, Utils.CheckOptions.FILE_ISFILE, Utils.CheckOptions.FILE_CANREAD);
```

Comment:
is FILE_EXISTS a subset of FILE_CANREAD?

Reply:
Yes, it is. Actually there are 20 places where this check is used. 
Here FILE_CANREAD covers both EXISTS && ISFILE

Reply:
if FILE_CANREAD is stronger than both other checks, should it be the only permission listed?

Reply:
Yes, here is enough to have just one the CANREAD check. OK  I will remove ISFILE, the first check will be alive just to catch that a file exists.